frontend/.gitignore

```diff
@@ -43,3 +43,6 @@ src/locales/scripts/wp-locales.json
 # To prevent accidentally adding a hardcoded robots.txt, see
 # /src/server-middleware/robots.js for the robots.txt file.
 src/static/robots.txt
+
+# Sentry Config File
+.env.sentry-build-plugin
```

Comment:
What is the reason for this being named `.env.sentry-build-plugin`? Could it be similar to the env setup for other other packages so that we can reuse the existing workflow (`.env` file created from `env.template` using the [`env` recipe](https://github.com/WordPress/openverse/blob/b0ac47d52ca9ee8e6350c7eaf9672e6f54f7c78d/justfile#L211-L212) etc.)?

Reply:
This was added by the Sentry module wizard. I added the env variable to the GitHub repository, so it's not really necessary anymore.